src/main/java/org/jabref/gui/fieldeditors/EditorValidator.java

```diff
@@ -0,0 +1,17 @@
+package org.jabref.gui.fieldeditors;
+
+import org.jabref.Globals;
+import org.jabref.preferences.JabRefPreferences;
+
+import de.saxsys.mvvmfx.utils.validation.ValidationStatus;
+import de.saxsys.mvvmfx.utils.validation.visualization.ControlsFxVisualizer;
+
+public class EditorValidator {
+
+    public static void configureValidation(ValidationStatus status, EditorTextArea area) {
+        if (Globals.prefs.getBoolean(JabRefPreferences.VALIDATE_IN_ENTRY_EDITOR)) {
```

Comment:
Can you please not use `Globals.prefs`. I really tried hard to make the JavaFX stuff free of `Globals`. You can simply pass a preference instance to the editor (similar how it is done in the OwnerEditor) and then to this method here (mabye as `new EntryValidator(preferences).configureValidation(...))`).

Reply:
Done! Good point also, we should thrive to keep this independent from `Globals`. I even spotted one more usage in `FieldEditors` that you seem to have overlooked ;-)